src/pages/go/quickstart.mdx

```diff
@@ -29,44 +29,35 @@ GOBIN in PATH
 
 <Steps>
 
-
 ### Import the library in your code
 
-In the main entrypoint of your program, you need to both add package
+In the main entrypoint of your program, add the package:
 
-```go
+```go filename="main.go"
 import (
-	autometrics "github.com/autometrics-dev/autometrics-go/pkg/autometrics/prometheus"
+	autometrics "github.com/autometrics-dev/autometrics-go/pkg/autometrics/otel"
 )
 ```
 
-And then in your main function initialize the metrics
+And then in your main function initialize the metrics:
 
-```go
-	// Everything in BuildInfo is optional.
-	// You can also use any string variable whose value is
-	// injected at build time by ldflags.
+```go filename="main.go"
 	autometrics.Init(
 		nil,
 		autometrics.DefBuckets,
-		autometrics.BuildInfo{
-			Version: "0.4.0",
-			Commit: "anySHA",
-			Branch: "",
-		},
 	)
```

Comment:
The build info argument is missing now (can be `autometrics.BuildInfo{}`), and now that it's using `otel` instead of prometheus, the signature of the `Init` function changed, the first argument is not a pointer (`nil`) anymore, but needs to be a ["meter scope"](https://opentelemetry.io/docs/specs/otel/glossary/#instrumentation-scope) as a string

It's modified accordingly in the otel example

https://github.com/autometrics-dev/autometrics-go/blob/3f44dd856f4a3922fe3ab0070bceafad73a8a5ee/examples/otel/cmd/main.go#L31-L39